app/controllers/ems_common.rb

```diff
@@ -561,6 +563,10 @@ def button
     end
   end
 
+  def provider_documentation_url
+    "http://manageiq.org/documentation/getting-started/#adding-a-provider"
```

Comment:
Each EMS model can maintain its own url, defining the same method inside a specific provider will override this method, this url is for a generic provider.